MoneySplitter.Win10/CustomControls/TransctionsControl.xaml.cs

```diff
@@ -0,0 +1,24 @@
+﻿using MoneySplitter.Models;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace MoneySplitter.Win10.CustomControls
+{
+    public sealed partial class TransctionsControl : UserControl
+    {
+        public TransactionModel ViewModel
+        {
+            get { return (TransactionModel)GetValue(ViewModelProperty); }
+            set { SetValue(ViewModelProperty, value); }
+        }
+        public static readonly DependencyProperty ViewModelProperty = DependencyProperty.Register(
```

Comment:
Insert empty line here to separate different parts of logic

Reply:
add empty lines